ChangeLog

```diff
@@ -1,3 +1,29 @@
+2017-07-28  Leah South  <leah.south@hdr.qut.edu.au>
+
+    * DESCRIPTION: Setting LazyData to true.
+    * inst/sampleData/pf-data.csv: Removed.
+    * data/pfdata.rda: Replaces pf-data.csv.
```

Comment:
I would prefer if you restored the csv file in the package.

If you think you must have an .rda, then keep the data file in, sat, `inst/rawdata`.